### PR TITLE
zh-translation: docs/root/intro/arch_overview/security/external_deps.rst

### PR DESCRIPTION
**Pull request by scilla0531**
ref: https://github.com/cloudnativeto/envoy/issues/37

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/intro/arch_overview/security/external_deps.rst [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-External dependencies
+外部依赖
 =====================
 
-Below we enumerate the external dependencies that may be linked into the Envoy binary. We exclude
-dependencies that only are used in CI or developer tooling above.
+下面列举了可能被链接到 Envoy 二进制文件中的外部依赖项，排除了那些只在 CI 或开发者工具中使用的依赖项。
 
 Data plane (core)
 -----------------
```
Comment (helight) [inline docs/root/intro/arch_overview/security/external_deps.rst:9]: 这些也需要翻译吧

**Review by helight — COMMENTED** [review]
上面的没问题，下面这些应该也是需要翻译的，你看看

**Comment by scilla0531** [comment]
@helight 修改已经提交，请查阅

File: docs/root/intro/arch_overview/security/external_deps.rst [hunk 2]
```diff
@@ -1,47 +1,46 @@
...
+下面列举了可能被链接到 Envoy 二进制文件中的外部依赖项，排除了那些只在 CI 或开发者工具中使用的依赖项。
 
-Data plane (core)
+数据平面 （core）
 -----------------
 
 .. include:: external_dep_dataplane_core.rst
 
-Data plane (extensions)
+数据平面 （extensions）
```
Comment (majinghe) [inline docs/root/intro/arch_overview/security/external_deps.rst]: 这个 extention 和core 是不是也应该翻译一下。比如 数据平面（核心），数据平面（扩展）。下同。

**Pull request merged**